src/app/georeferencer/qgsgcplist.h

```diff
@@ -29,7 +29,7 @@ class QgsGCPList : public QList<QgsGeorefDataPoint *>
     QgsGCPList() = default;
     QgsGCPList( const QgsGCPList &list );
 
-    void createGCPVectors( QVector<QgsPointXY> &mapCoords, QVector<QgsPointXY> &pixelCoords );
+    void createGCPVectors( QVector<QgsPointXY> &mapCoords, QVector<QgsPointXY> &pixelCoords, const QString &wktProj = QString() );
```

Comment:
What your passing on is the QgsCoordinateReferenceSystem's authid(), calling this variable wktProj is misleading.

Reply:
should I pass a CRS directly instead here also? I was on the fence, initially it was a wkt that was passed but changed to authid to debug the faulty crs widget.

Reply:
Yes, CRS object here would be advised here too.

Reply:
done